components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthServiceComponent.java

```diff
@@ -556,4 +571,27 @@ protected void unregisterConfigurationManager(ConfigurationManager configuration
         }
         OAuthComponentServiceHolder.getInstance().setConfigurationManager(null);
     }
+
+    @Reference(
+            name = "action.execution.service",
+            service = ActionExecutorService.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unregisterActionExecutorService"
+    )
+    protected void registerActionExecutionService(ActionExecutorService actionExecutorService) {
+
+        if (log.isDebugEnabled()) {
```

Comment:
we don't need to check ```isDebugEnabled```, cause the log gonna print doesn't have any string concatenation. So checking isDebug enabled is unnecessary

Reply:
ack. will remove

Reply:
Fixed